app/javascript/components/Driver/CheckpointWindow.tsx

```diff
@@ -20,17 +20,17 @@ const CheckpointWindow:
 
     const statusChange = () => {
       if (status) {
-        httpClient.route.rollback({ ids: id }).then(() => {
-          update_checkpoint_status(wayID);
+        httpClient.route.rollback({ ids: id }).then((response) => {
+          update_checkpoint_status(wayID,response.data);
           handleClose();
         });
       } else setActiveModal(true);
     };
     const handleSubmit = (values) => {
       Object.assign(values, { ids: id });
       httpClient.route.passCh(values)
-        .then(() => {
-          update_checkpoint_status(wayID);
+        .then((response) => {
+          update_checkpoint_status(wayID,response.data);
```

Comment:
After inputing pass date and click "save" button checkpoint is not updating on front-end side though update hits backend. i guess it is because controller changes.